compiler/testData/codegen/box/enum/bigEnum.kt

```diff
@@ -0,0 +1,412 @@
+// TARGET_BACKEND: JVM
+// IGNORE_BACKEND: JVM
```

Comment:
Let me know where old backend creates enum's (special) members.

Reply:
Since we're going to enable JVM IR by default since 1.5, we've actually frozen any work on the old backend. So it's OK to support this only in the new backend.

Reply:
Sounds good, thanks for the confirmation.